### DISCUSSION TITLE
请问Playcover里安装的ipa应用的数据可以储存到移动硬盘吗

### DISCUSSION THREAD

**Discussion by zxjzxj0010** [PlayCover/PlayCover#1691] [General]
请问Playcover里安装的ipa应用的数据可以储存到移动硬盘吗
Could the data of the IPA applications installed in Playcover be stored on a mobile hard drive?

**Comment by Micro-ATP** [comment]
你可以参考这个：https://github.com/PlayCover/PlayCover/discussions/1712
